CameraServer/Helpers/Camera.cs

```diff
@@ -34,10 +34,16 @@ public async Task<CameraImage> GetImageAsync()
 
         public bool IsValidToken(string token)
         {
-            if(information == null || information.Token == null)
+            if (information == null || information.Token == null)
                 return false;
 
             return TokenHelper.GetToken(information.Id) == information.Token;
         }
+
+        public async void SetPreview()
```

Comment:
Bryter naming convention, borde heta SetPreviewAsync(), igen samma artikel men med en specifik sektion länkad om du inte orkar läsa hela: https://learn.microsoft.com/en-us/dotnet/csharp/asynchronous-programming/async-scenarios#important-info-and-advice